### DISCUSSION TITLE
File Manager - New Folder auto focus on Folder Name

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#3957] [Feature Requests]
### Suggest an idea for ExpressionEngine

When creating a New Folder in the File Manager, would it be possible to auto-focus on the Folder Name field?

**Comment by intoeetive** [comment]
@Yulyaswan is that something that we can do? I think focusing input in modal (if present) makes sense UX-wise

**Comment by Yulyaswan** [comment]
PR - https://github.com/ExpressionEngine/ExpressionEngine/pull/3961


https://github.com/ExpressionEngine/ExpressionEngine/assets/23382425/16d84ce0-affc-469c-a0ce-841bf531eedb

**Comment by gregbowers** [comment]
implement auto-focus on the Folder Name field when creating a new folder. This small adjustment enhances user-friendliness, saves time, and improves accessibility, aligning with the goal of a more efficient and user-centric platform.
